src/templates/varTemplates.ts

```diff
@@ -33,12 +35,16 @@ export class VarTemplate extends BaseExpressionTemplate {
     const buildVarName = (name: string) => name && _.lowerFirst(name)
 
     if (ts.isNewExpression(node)) {
-      return buildVarName(this.inferNewExpressionVar(node))
+      return [buildVarName(this.inferNewExpressionVar(node))]
     } else if (ts.isCallExpression(node)) {
       const methodName = this.getMethodName(node)
-      const matches = VarTemplate.MethodCallRegex.exec(methodName)
+      const name = VarTemplate.MethodCallRegex.exec(methodName)?.groups?.name
+      const cleanerVariant = name?.replace(VarTemplate.CleanNameRegex, '')
+      if (!name) {
+        return
+      }
 
-      return buildVarName(matches?.groups?.name)
+      return [...cleanerVariant && cleanerVariant !== name ? [cleanerVariant] : [], name].map(buildVarName)

```

Comment:
As long as this works fine, it's not very readable and easily maintainable. How about simplifying that as follows:
```js
      return _.uniq([cleanerVariant, name]).filter(x => x).map(buildVarName)
// or even better without Lodash, just ES6
      return [...new Set([cleanerVariant, name])].filter(x => x).map(buildVarName)
```
It will be much easier to extend when we come up with another idea of more than two suggestions.